crates/katana/node/src/config/dev.rs

```diff
@@ -31,6 +32,11 @@ pub struct DevConfig {
     ///
     /// These are the prices that will be used for calculating the gas fee for transactions.
     pub fixed_gas_prices: Option<FixedL1GasPriceConfig>,
+
+    /// L1 gas oracle worker task configuration for real time gas sampling.
+    ///
+    /// If sampling disabled, the system falls back to the hardcoded gas values.
+    pub l1_worker: Option<GasPriceWorkerConfig>,
```

Comment:
```suggestion
```

we dont need a dedicated config for the gas oracle worker just yet, and even if we do, it shouldn't belong under `DevConfig`.